models/balancer/ethereum/balancer_ethereum_vebal_balances_day.sql

```diff
@@ -0,0 +1,133 @@
+WITH base_locks AS (
```

Comment:
most, if not all, models contain a config() block at the top with at minimum an 'alias' property applied, which allows you to specify the view name. without the alias, it'll default to the file name:
balancer_ethereum.balancer_ethereum_vebal_balances_day

the schema name will come from the dbt project yaml file, mentioned in prior comment

i'd assume you'd want to alias to:
balancer_ethereum.vebal_balances_day

Reply:
example: https://github.com/duneanalytics/spellbook/blob/main/models/prices/prices_tokens.sql

you can leave the schema property out if you apply correctly in dbt project file